2/test.js

```diff
@@ -0,0 +1,46 @@
+// Import the mount() method from the test utils
+// and the component you want to test
+import { mount } from '@vue/test-utils'
+import ListItem from './ListItem'
```

Comment:
Call this file `ListItem.spec.js` so every developer immediately knows which component it refers to.